lib/apiservers/portlayer/swagger.json

```diff
@@ -593,6 +593,68 @@
 					}
 				}
 			},
+			"post": {
+				"description": "Join an image to a container",
+				"operationId": "ImageJoin",
+				"tags": [
+					"storage"
+				],
+				"consumes": [
+					"application/json",
+					"application/octet-stream"
+				],
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"name": "store_name",
+						"type": "string",
+						"in": "path",
+						"required": true
+					},
+					{
+						"name": "id",
```

Comment:
should we rename this to layerid?  I know it's very specific to docker, but we've been following a loose convention of setting id to the object that we're acting on.  Here, we doing ImageJoin, but the id is weirdly layerid.  If we follow our loose convention, id would be something like the container or handle.  Since it's not one of those, maybe we should rename this from id to layerID or shove it into the config parameter.

Reply:
The layerID _is_ the object we're acting on - that's the ID of the image that we're joining to the container.